Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -39,11 +38,10 @@ public class InstallationManager : IInstallationManager
         /// <summary>
         /// The completed installations
         /// </summary>
-        private ConcurrentBag<InstallationInfo> CompletedInstallationsInternal { get; set; }
+        private ConcurrentBag<InstallationInfo> _completedInstallationsInternal;
 
-        public IEnumerable<InstallationInfo> CompletedInstallations => CompletedInstallationsInternal;
+        public IEnumerable<InstallationInfo> CompletedInstallations => _completedInstallationsInternal;
 
-        #region PluginUninstalled Event
```

Comment:
Why the hate on regions?

Reply:
Useless, ugly and extra bloat on my screen ;)
The analyzers and SO agree with me ;)